packages/react-native/ReactNativeApi.d.ts

```diff
@@ -5586,7 +5615,7 @@ declare type TouchEventProps = {
 declare type TransformsStyle = ____TransformStyle_Internal
 declare interface TurboModule extends DEPRECATED_RCTExport<void> {}
 declare namespace TurboModuleRegistry {
-  export { get, getEnforcing }
+  export { get_2 as get, getEnforcing }
```

Comment:
What is the difference between `get `and `get_2`?

Reply:
HI. No difference, just a name conflict of two `get` function.